static/i18n/en/app.json

```diff
@@ -2887,16 +2887,16 @@
       "description": "An unexpected error occurred. Please try again. ({{message}})"
     },
     "DeviceSocketNoBulkStatus": {
-      "title": "Oops, connection failed",
-      "description": "Please try again"
+      "title": "The connection failed",
+      "description": "Please try again."
     },
     "DisconnectedDevice": {
-      "title": "Oops, device connection lost",
-      "description": "Please try again"
+      "title": "Sorry, it looks like your device was disconnected",
```

Comment:
Present perfect is more appropriate in that case imo, correct me if I'm wrong @dasilvarosa.
`was` -> `has been`